src/EncryptUtility1.java

```diff
@@ -0,0 +1,209 @@
+interface EncryptionUtility {
```

Comment:
Does this compile? Java doesn't usually let you have two "roots" in one file. They usually need to be nested.